docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -134,18 +134,32 @@ testcafe "saucelabs:Chrome@52.0:Windows 8.1" tests/sample-fixture.js
 
 ### Starting browser with arguments
 
-If you need to pass arguments for the specified browser, write them right after browser alias. Surround the browser call and its arguments with quotation marks:
+If you need to pass arguments for the specified browser, write them right after browser alias. Surround the browser call and its arguments with quotation marks. On Windows, use double quotation marks.
```

Comment:
I found that double quotes don't work well with Windows PowerShell. This paragraph should be about Windows cmd only. Like:


In Windows cmd.exe (the default Command Prompt), use double quotation marks